pom.xml

```diff
@@ -16,6 +16,8 @@
     <module>runner</module>
     <module>bom</module>
     <module>build-parent</module>
+    <module>ingress</module>
+    <module>infra</module>
```

Comment:
Is the `infra` module the Fleet Shard?

Reply:
Nope, it is a module that contains DTOs and other common utilities. The `shard` module is called `shard`